manage/config-dialog.js

```diff
@@ -286,7 +313,38 @@ function configDialog(style) {
   }
 
   function updateVarOnChange() {
-    this.va.value = this.type !== 'checkbox' ? this.value : this.checked ? '1' : '0';
+    if (this.type === 'range') {
+      this.va.value = Number(this.value);
+      $('.current-value', this.closest('.config-range')).textContent = this.va.value + (this.va.units || '');
+    } else if (this.type === 'number') {
+      // clamp stored value
+      this.va.value = clampValue(this.value, this.va);
```

Comment:
It should be `Number(this.value)`. `input.value` is always a string.

Reply:
DOH, fixed.